auth/src/main/java/org/aerogear/android/ags/auth/impl/OIDCTokenAuthenticatorImpl.java

```diff
@@ -26,4 +48,66 @@ public Principal authenticate(ICredential credential) throws AuthenticationExcep
         // This authenticator can't manage this type of credential
         return null;
     }
+
+    /**
+     * Logs out a user from openID connect server.
+     *
+     * @param principal principal to be logged out.
+     */
+    public void logout(final Principal principal) {
+        // Get user's identity token
+        OIDCCredentials credentials = (OIDCCredentials) ((IUserPrincipal) principal).getCredentials();
```

Comment:
Is that copied from some example? Seen that code before

Reply:
I haven't had a proper look but I don't think this or anything similar is in our SDK at the moment no